internal/allocator/allocator.go

```diff
@@ -72,25 +72,37 @@ func poolCount(p *config.Pool) int64 {
 	for _, cidr := range p.CIDR {
 		o, b := cidr.Mask.Size()
 		sz := int64(math.Pow(2, float64(b-o)))
+		avoidedFirst, avoidedLast := false, false
```

Comment:
Present tense for these: avoidFirst, avoidLast ?
(super nit)

Reply:
Done.